stories/module-adsense-components.stories.js

```diff
@@ -411,6 +412,266 @@ generateReportBasedWidgetStories( {
 	},
 } );
 
+const dataModuleTopEarningPagesWidget = [
```

Comment:
@johnPhillips we shouldn't use hard-coded fixtures for Analytics reports anymore. Let's use the generator function as we do it for Analytics widgets: https://github.com/google/site-kit-wp/blob/07e6331943137bb8dc8318f0a06ca4b68ca0c0c0/stories/module-analytics-components.stories.js#L426-L460